packages/create-amplify/src/logger.ts

```diff
@@ -1,18 +1,29 @@
 import yargs from 'yargs';
+import * as os from 'os';
 
 /**
  * A logger that logs messages to the console.
  */
 export class Logger {
   /**
-   * Creates a new Logger instance.
+   * Properties for ellipsis animation
+   */
+  private timer: ReturnType<typeof setTimeout>;
+  private refreshRate: number;
+  private animateMessage: string;
+
+  /**
+   * Creates a new Logger instance. Injecting stdout for testing.
    * @param console The console to log to.
    * @param minimumLogLevel The minimum log level to log.
    */
   constructor(
     private readonly console: Console = global.console,
-    private readonly minimumLogLevel: LogLevel = LogLevel.INFO
-  ) {}
+    private readonly minimumLogLevel: LogLevel = LogLevel.INFO,
+    private readonly stdout = process.stdout
```

Comment:
Does `console.log` not write to `stdout`? We can perhaps get rid of `console`?

Reply:
From what I found, `console.log` uses `stdout.write` under the hood with formatting and ending the log with a newline. Ending the log with a newline is what we don't want for the animation so I think we can get rid of `console` and use `stdout.write` and end with a newline. This will have the same results as `console.log` since we don't use console's formatting at the moment.

Reply:
Done in [fb89cac](https://github.com/aws-amplify/amplify-backend/pull/595/commits/fb89cacbc4748373aa55231303d37aa8d05dccbc)